lambdas/service/app.py

```diff
@@ -51,11 +51,19 @@
 # how-to-write-docstring-for-url-parameters
 
 
-def _get_pagination(current_request):
+sort_defaults = {
+    'files': ('fileName', 'asc'),
+    'specimens': ('specimenId', 'asc'),
+    'projects': ('projectTitle', 'asc'),
+}
+
+def _get_pagination(current_request, entity_type):
+    default_sort = sort_defaults[entity_type][0]
```

Comment:
```suggestion
    default_sort, default_order = sort_defaults[entity_type]
```

Reply:
Ah yes, that is cleaner. I'll make that change now.